twindle-web/style.css

```diff
@@ -52,7 +50,7 @@ body::-webkit-scrollbar-thumb{
 /* Navigation Bar */
 
 header {
-  display: flex;
+  display: inline;
```

Comment:
Why did you remove the display: flex property, & used display:inline. Please also explain this too what it result too. Flexbox is a great way to make the header responsive

Reply:
![Capture](https://user-images.githubusercontent.com/63865514/101989933-a3da0700-3cc9-11eb-9a6a-2901bb20ace3.PNG)